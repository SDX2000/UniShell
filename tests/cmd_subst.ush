echo "================"
echo $SCRIPT_NAME
echo "================"


#sanity check
echo a
echo "hello"
echo a b
echo "hello" "world"

#
echo $(echo a)

#
echo $(echo a) $(echo b)
echo $(echo "hello") $(echo "world")

#Subst inside quotes

echo "$(echo a)"
