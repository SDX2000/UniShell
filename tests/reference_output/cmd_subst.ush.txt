================
cmd_subst.ush
================
a
hello
a b
hello world
a
a b
hello world
a
abc
1
+1
-1
1
-1
1
1
-1
-0.11
11.0
1
1
-1
-0.11
11.0
$(echo "a")
